cmd/bootstrap/main.go

```diff
@@ -83,7 +83,7 @@ func main() {
 	defer esClient.Close()
 
 	// Create new auth grpc client api
-	auth, authHandler, err := authClient.Setup(envPrefix, svcName, cfg.JaegerURL)
+	auth, authHandler, err := authClient.Setup(envPrefix, cfg.JaegerURL, svcName)
```

Comment:
why not change `authClient.Setup` to accept `svcName` first then `JaegerURL`?

Reply:
Some of the implementations had the correct order of parameters

Reply:
That was only one service `things`

Reply:
@rodneyosodo 
In Auth gRPC client its defined as 
```go
func Setup(envPrefix, jaegerURL, svcName string)
```
https://github.com/mainflux/mainflux/blob/master/internal/clients/grpc/auth/client.go#L19

In things gRPC client defined as 
```go
func Setup(envPrefix, jaegerURL string)
```
https://github.com/mainflux/mainflux/blob/master/internal/clients/grpc/things/client.go#L19

Because of this alignment, `svcName` went last

Reply:
I'd also prefer `svc` name before JaegerURL.  Please fix it in `/internal/clients/grpc` as well.